src/vs/workbench/browser/parts/editor/editorDropTarget.ts

```diff
@@ -328,16 +329,20 @@ class DropOverlay extends Themable {
 							return;
 						}
 
-						// Open in target group
-						const options = fillActiveEditorViewState(sourceGroup, draggedEditor.editor, {
-							pinned: true,										// always pin dropped editor
-							sticky: sourceGroup.isSticky(draggedEditor.editor),	// preserve sticky state
-						});
+						const editors = draggedEditors.map(de => (
```

Comment:
Whats this change about? And is `de` a typo?

Reply:
de = draggedEditor